### DISCUSSION TITLE
Google Play store

### DISCUSSION THREAD

**Discussion by prasitsuankaeomanee** [community/community#78054] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

,กลุ่มเบต้าและการอัปเดตบางทีให้อัตโนมัติแต่ระบบปฎิบัติการเวอร์ชั่นไม่ถึงเช่น Chrome ให้อัปเดตเป็น Google Chrome ระบบปฏิบัติการจองโทรศัพท์ไม่ถึงใช้ไม่ได้จะให้อัปเดตระบบทําไมในเมื่อเวอร์ชั่นเก่าก็ใช้ได้ดีอยู่แถมยังเป็น Google LLC ใน Play store ก็กลายเป็น Google Play store แถมยังมีรุ่นเบต้า ถ้าเครื่องมันรวนระบบผิดปกติใครรับผิดชอบเพราะว่ามีบางเครื่องใช้ระบบปฏิบัตินักพัฒนาซอฟต์แวร์ รุ่น เบต้า ที่ไม่เสถียรแถมยังเอามาใช้ รายงานไปก็ไม่เห็นมีอะไรเปลี่ยนแปลงซักเท่าไหร่รึว่าไม่สนใจนักพัฒนาซอฟต์แวร์ปล่อยตามแต่เรื่องเอากันแต่เงินแล้วนพัฒนาระบบเสร็จแม่งก็ยึดทําไงดีครับ ช่วยพิจารณาด้วยนะครับ ขอบคุณครับผม

**Comment by queenofcorgis** [comment]
Hi @prasitsuankaeomanee ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Google and their products. The GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
